src/kvstore/RocksEngineConfig.cpp

```diff
@@ -196,6 +224,9 @@ rocksdb::Status initRocksdbOptions(rocksdb::Options &baseOpts) {
     }
     baseOpts.table_factory.reset(NewBlockBasedTableFactory(bbtOpts));
     baseOpts.create_if_missing = true;
+    if (FLAGS_enable_prefix_bloom_filter) {
+        baseOpts.prefix_extractor.reset(new GraphPrefixTransform(FLAGS_prefix_length));
```

Comment:
Do you need to check the `FLAGS_prefix_length` is the specified size ? Maybe setting it as 12 or 16 is OK ？

Reply:
I think it is ok for users to config prefix length. As my data set is so big, I close whole key filters by default, all requests rely on prefix bloom filter heavily.

Reply:
> As my data set is so big, I close whole key filters by default.

Maybe we should make this as the default behavior? According to **BlockBasedTable::FullFilterKeyMayMatch**, prefix bloom filer seems to be enough for the nebula.

Reply:
Agree, prefix bloom filter is more useful than full key filter, with much less memory consumption.